src/similarity.rs

```diff
@@ -68,12 +77,31 @@ where
         word3: &str,
         limit: usize,
     ) -> Option<Vec<WordSimilarity>> {
-        self.analogy_by(word1, word2, word3, limit, |embeds, embed| {
-            embeds.dot(&embed)
-        })
+        self.analogy_by(
+            word1,
+            word2,
+            word3,
+            limit,
+            [false, false, false],
```

Comment:
I am a bit confused. I thought being masked would mean that the word is not included in the result set. Then the 'traditional' analogy implementation should have `[true, true, true]` as the mask?

Reply:
I initially had a failing CI build because I confused myself with this, too. 

I was thinking of masks as something like we use in tf models. true (or 1) for active steps and false (or 0) for inactive steps. Similarly here, false for those words that are inactive.